osf/utils/migrations.py

```diff
@@ -390,6 +390,39 @@ def create_schema_blocks_for_question(state, rs, question, sub=False):
         split_options_into_blocks(state, rs, question, schema_block_group_key)
 
 
+
+def create_schema_blocks_for_simple_schema(schema):
+    from osf.models import RegistrationSchemaBlock
+    schema_block_group_key = ''
+    page_num = 0
+    q_num = 1
```

Comment:
looks like these have become unused and the logic is in your schemablock generator?